docs/additional-configs.md

```diff
@@ -82,9 +82,6 @@ use a `ConfigMap` with the name `config-artifact-bucket` and the following attri
 
 **Important:** Configure your bucket's retention policy to delete all files after your `Tasks` finish running.
 
-**Note:** You can only use an S3 bucket located in the `us-east-1` region. This is a limitation of [`gsutil`](https://cloud.google.com/storage/docs/gsutil) running a `boto` configuration behind the scenes to access the S3 bucket.
```

Comment:
Why removing this note?

Reply:
Thanks for catching, I think instead this whole section shall be removed for `##Configuring PipelineResource storage`